packages/webapp/src/util/siteMapConstants.ts

```diff
@@ -16,6 +16,13 @@
 // Type of string has the dual purpose of being used in routes parameter pathing
 //   as well as accomodating UUID types.
 
+// Animals
+
+export const ANIMALS_URL = '/animals';
+export const ANIMALS_HOME_URL = '/animals/inventory';
```

Comment:
NIT, but I'd call this one `ANIMALS_INVENTORY_URL` since it's not really a home in the sense that we never default to it. With finances I think we ended up with that home word because with the previous navigation clicking on the main Finances menu item lead you to what is now the Transactions section (but it should probably be updated eventually as well).

Reply:
sure its done